### DISCUSSION TITLE
API Key follows "Default Alias Format"

### DISCUSSION THREAD

**Discussion by jack-fin** [anonaddy/anonaddy#436] [Feature Requests]
Hello,

Really like AnonAddy, I use it with Bitwarden.

Right now Bitwarden interacts with the AnonAddy API and does not allow you to specify options such as "Default Alias Format" being Random Words or Random Characters. Since it appears the API default is "Random Characters" this is what Bitwarden produces.

It would be nice that if the API call does not specify what email format to use, AnonAddy would default to my "Default Alias Format" of Random Words (instead of Random Characters)

**Comment by willbrowningme** [comment]
Thanks, I've just pushed an update that adds this. Now if no alias `format` is present in the API request the user's `default_alias_format` will be used.
